pkg/scheduler/eventhandlers_test.go

```diff
@@ -334,3 +341,83 @@ func TestPreCheckForNode(t *testing.T) {
 		})
 	}
 }
+
+// test for informers of resources we care about is registered
+func TestAddAllEventHandlers(t *testing.T) {
+	tests := []struct {
+		name               string
+		gvkMap             map[framework.GVK]framework.ActionType
+		staticInformerNum  int
+		dynamicInformerNum int
+	}{
+		{
+			name:               "default handlers in framework",
+			gvkMap:             map[framework.GVK]framework.ActionType{},
+			staticInformerNum:  2,
+			dynamicInformerNum: 0,
+		},
+		{
+			name: "add GVKs handlers defined in framework dynamically",
+			gvkMap: map[framework.GVK]framework.ActionType{
+				"Pod":                               framework.Add | framework.Delete,
+				"PersistentVolume":                  framework.Delete,
+				"storage.k8s.io/CSIStorageCapacity": framework.Update,
+			},
+			staticInformerNum:  4,
+			dynamicInformerNum: 0,
+		},
+		{
+			name: "add GVKs handlers defined in plugins dynamically",
+			gvkMap: map[framework.GVK]framework.ActionType{
+				"daemonsets.v1.apps": framework.Add | framework.Delete,
+				"cronjobs.v1.batch":  framework.Delete,
+			},
+			staticInformerNum:  2,
```

Comment:
would it be possible to make these `map[reflect.type]bool` instead of mere counters?
But if it's too much work you can leave it.

Reply:
ok